packages/ui/src/components/va-checkbox/VaCheckbox.vue

```diff
@@ -153,7 +156,7 @@ export default defineComponent({
       labelStyle,
       inputStyle,
       computedIconName,
-      computedIconColor,
```

Comment:
`computedIconColor ` can be safety removed from this component. Computed still declared above.